packages/components-js/tests/e2e/puppeteer/specs/table.e2e.ts

```diff
@@ -256,29 +207,6 @@ describe('accessibility', () => {
     expect(await getAttribute(caption, 'id'), 'caption id').toBe('caption');
   });
 
-  it('should expose correct accessibility tree of scroll area', async () => {
```

Comment:
Please verify with Ole if a11y is still good.

Reply:
I'll ask Ole tomorrow